frontend/components/feedback-task/field/TextField.component.vue

```diff
@@ -18,21 +18,26 @@
         </BaseActionTooltip>
       </div>
       <div class="content-area --body1">
-        <div v-if="!useMarkdown" v-text="fieldText" />
-        <RenderMarkdownBaseComponent v-else :markdown="fieldText" />
+        <div v-if="!useMarkdown" v-text="text" />
```

Comment:
If the highlight won't work for non markdown rendering, why we're passing the `text` instead the origin `fieldText`?